src/org/zaproxy/zap/extension/ascanrules/TestSQLInjection.java

```diff
@@ -692,93 +690,124 @@ public void scan(HttpMessage msg, String param, String origParamValue) {
 					if (this.debugEnabled) {
 						log.debug("The parameter value [" + origParamValue + "] is of type Integer");
 					}
+					// This check is implemented using two variant PLUS(+) and MULT(*)
+					// PLUS variant check the param value "3-2" gives same result and param value "4-2" gives different result if original param value is 1
+					// PLUS variant check the param value "2/2" gives same result and param value "4/2" gives different result if original param value is 1
```

Comment:
It might help if the text is slightly more specific "...gives the same result as the original request..."